src/cli/cli_dataset.cpp

```diff
@@ -205,12 +205,17 @@ template <> otError Dataset::Process<Cmd("activetimestamp")>(Arg aArgs[])
     {
         if (sDataset.mComponents.mIsActiveTimestampPresent)
         {
-            OutputLine("%lu", sDataset.mActiveTimestamp);
+            OutputLine("%lu", sDataset.mActiveTimestamp.mSeconds);
         }
     }
     else
     {
-        SuccessOrExit(error = aArgs[0].ParseAsUint64(sDataset.mActiveTimestamp));
+        uint64_t activeTimestampSeconds;
+
+        SuccessOrExit(error = aArgs[0].ParseAsUint64(activeTimestampSeconds));
```

Comment:
what if activeTimestampSeconds exceeds 48 bits?

Reply:
`0xffffffffffff` seconds are more than 8921556 years, I think it doesn't worth handling.